src/theme/ItaliaTheme/Blocks/_bandiInEvidenceTemplate.scss

```
@@ -51,14 +51,34 @@
   }
 
   // tablet
-  @media (min-width: 586px) and (max-width: 1367px) {
-    .bandi-in-evidence-cards-wrapper {
+  .bandi-in-evidence-cards-wrapper {
+    @media (min-width: 767px) and (max-width: 1367px) {
```

Usare variabili tipo 
```suggestion
    @media (min-width: #{map-get($grid-breakpoints, md)}) and (max-width: #{map-get($grid-breakpoints, lg)}) {
```